src/common/Microsoft.DotNet.UpgradeAssistant.Abstractions/IAnalyzeResultProvider.cs

```diff
@@ -0,0 +1,13 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Microsoft.DotNet.UpgradeAssistant
+{
+    public interface IAnalyzeResultProvider
```

Comment:
May be good to put an obsolete tag on as a warning that things may change

Reply:
Added